File: dotnet/src/Pactman.NonprofitCheckPlus/Exceptions/PactmanServerException.cs
namespace Pactman.NonprofitCheckPlus.Exceptions
{
    /// <summary>HTTP 5xx. The API failed to process the request.</summary>
    public sealed class PactmanServerException : PactmanApiException
    {
        /// <summary>Initializes the exception from response metadata.</summary>
        /// <param name="message">Human-readable description of the failure.</param>
        /// <param name="init">The metadata the transport collected from the response.</param>
        public PactmanServerException(string message, ApiErrorInit init)
            : base(message, init, ErrorCategory.Server)
        {
        }
    }
}

File: java/sdk/src/main/java/org/pactman/nonprofitcheckplus/exceptions/PactmanServerException.java
package org.pactman.nonprofitcheckplus.exceptions;

/** HTTP 5xx. */
public class PactmanServerException extends PactmanApiException {

    private static final long serialVersionUID = 1L;

    /**
     * Creates the exception.
     *
     * @param message the message to surface.
     * @param init    the response metadata.
     */
    public PactmanServerException(String message, ApiErrorInit init) {
        super(message, init, ErrorCategory.SERVER);
    }
}
